starter-code/index.html

```diff
@@ -2,30 +2,54 @@
 <html lang="en">
 <head>
   <title>Instagram</title>
+  <link rel="stylesheet" href="./css/main.css">
 </head>
 <body>
-  Instagram
 
-  Capture and Share the World's Moments
+  <div class="everything">
+    <div class="phones">
+      <img src="./images/phones.png" alt="phones-pic">
+    </div>
 
-  Instagram is a fast, beautiful and fun way to share your life with friends and family.
+    <section class="main">
+      <img class="insta-logo" src="./images/brand.png" alt="Instagram Logo">
+      <button>
+        <img class="home-logo" src="./images/home.png" alt="home-logo">
+        <p class="text-logo">Log in</p>
+      </button>
 
-  Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.
+    <div class="white-frame">
+      <h1>Capture and Share<br>
+      the World's Moments</h1>
 
-  Oh yeah, did we mention it's free?
+      <p>Instagram is a <strong>fast</strong>, <strong>beautiful</strong> and <strong>fun</strong> way to share your life with friends and family.<br><br>
 
-  Download on the App Store
-  Get it on Google Play
+      Take a picture or video, choose a filter to transform its look and feel, then post to
+    Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.<br><br>
 
-  About Us
-  Support
-  Blog
-  Press
-  API
-  Jobs
-  Privacy
-  Terms
-
-  &copy; 2014 Instagram
+    Oh yeah, did we mention it's free?</p>
+    
+      <div class="bottom-logo">
+        <img class="apple-logo" src="./images/badge-iphone.png" alt="apple-logo-app">
+        <img class="android-logo" src="./images/badge-android.png" alt="android-logo-app">
+      </div>
+    </div>
+    
+    </section>
+    
+    <footer>
```

Comment:
Same as bottom-logo

If you inspect the element you'll see it's fitting not propperly in the page